### ISSUE TITLE
Guide New Developers/Student 

### ISSUE THREAD

**Issue by Praashh** [hkirat/full-stack-assignment#125]
Hey Everyone Please Guide new student who want to learn more like me or other if anyone facing issue in order to complete assignment feel free to ask anything in this section.

**Comment by bharathkalyans** [comment]
There's no need of opening an issue for this right? 
Just go through everyone's PRs, read their code and directly comment on the line which you are not able to understand. Or simply copy their code and paste it to chatGPT and ask it to analyze. But I am pretty sure people will definitely answer your queries.

**Comment by ERENxJEAGER11** [comment]
May be @9520prashant  is right, there can be some folks who is in need even after some brainstorming. even though @bharathkalyans has a good point about chat GPT but connecting with developers of your or above level will always teach you something

**Comment by bharathkalyans** [comment]
True, I was just stating my opinion 😀.

**Comment by Praashh** [comment]
@bharathkalyans Yes bro you are right but sometime we need a person instead of AI etc.

**Comment by Praashh** [comment]
Hey Community I am taking the responsibility of solving issues of new folks if anyone have any query then ask with mentioning me.

**Comment by Baka-14** [comment]
hey i forked the repo cloned it made changes commited , now how do i make a pr to the repo ?

**Comment by bharathkalyans** [comment]
Follow these @Baka-14 
* Use this command to add upstream for the main repository 
```git remote add upstream https://github.com/hkirat/full-stack-assignment.git```
* You can verify the remote repositories using the command ```git remote -v```
* Assuming you have created a new branch for the feature(or any other change), use the following command ```git push upstream <branch-name>```
* Now open the same repository in your GitHub and you should see a dialog box on top of the repository asking you for a PR.
* Click on it add the necessary details and click on Submit.
I hope these steps are self-explanatory.

**Comment by tushar-vaishnav** [comment]
#390 
> 

- [ ] #- > ### ### ### ### [**_``_**](url)
/
```css

dasd
``` 


```python
print('tushar)
```

**Comment by jagadesh-angireddi** [comment]
Hey guys what to do you get when you search the local host 3001 after we run the code successfully

**Comment by bharathkalyans** [comment]
> Hey guys what to do you get when you search the local host 3001 after we run the code successfully

Can you be more elaborative??

**Comment by Ashutosh-510** [comment]
> 
I did same and understood every line of code by GPT of others solution.

**Comment by Ashutosh-510** [comment]
> There's no need of opening an issue for this right? Just go through everyone's PRs, read their code and directly comment on the line which you are not able to understand. Or simply copy their code and paste it to chatGPT and ask it to analyze. But I am pretty sure people will definitely answer your queries.

  I did same and understood every line of code by chatGPT .

**Comment by bharathkalyans** [comment]
> > There's no need of opening an issue for this right? Just go through everyone's PRs, read their code and directly comment on the line which you are not able to understand. Or simply copy their code and paste it to chatGPT and ask it to analyze. But I am pretty sure people will definitely answer your queries.
> 
> I did same and understood every line of code by chatGPT .

Great ✊🏼
